### PR TITLE
[4.0] configuration tab background

### PR DESCRIPTION
**Pull request by brianteeman**
Small tweak to the background colour of the active color to increase the contrast between active and non-active tabs

### before
image

### after
image

### CODE REVIEW

**Comment by C-Lodder** [comment]
You can simply add the following:

```
$nav-tabs-link-active-bg: rgba($black, .2);
```

To the template `variables.scss` file

**Comment by brianteeman** [comment]
Thanks @C-Lodder I updated this with your suggestion

File: administrator/templates/atum/scss/_variables.scss [hunk 1]
```diff
@@ -126,6 +126,7 @@ $dropdown-spacer:                  0;
 $nav-tabs-border-width:            0;
 $nav-tabs-border-radius:           0;
 $nav-tabs-link-active-bg:          rgba($black, .1);
+$nav-tabs-link-active-bg:          rgba($black, .2);
```
Comment (wilsonge) [inline administrator/templates/atum/scss/_variables.scss:128]: This is the same variable as directly above so we should delete the one above

Comment (brianteeman) [reply administrator/templates/atum/scss/_variables.scss:128]: oops

**Comment by brianteeman** [comment]
should be ok now @wilsonge

**Comment by wilsonge** [comment]
LGTM! Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**